packages/cli/src/commands/mock.ts

```diff
@@ -64,17 +28,14 @@ const mockCommand: CommandModule = {
       port,
       host,
       cors,
-      operations,
-      spec,
-    } = (parsedArgs as unknown) as CreatePrismOptions & {
-      multiprocess: boolean;
-      spec: string;
-    };
+      document,
+      reportViolations,
+    } = (parsedArgs as unknown) as CreateMockServerOptions;
 
     const createPrism = multiprocess ? createMultiProcessPrism : createSingleProcessPrism;
-    const options = { cors, dynamic, port, host, operations };
+    const options = { cors, dynamic, port, host, document, multiprocess, reportViolations };
```

Comment:
The `multiprocess` option has done its job and is no longer needed. However, ignore the comment if it adds too much spaghetti into the codebase.

Reply:
Yeah — basically I need to go through and modify different statement to be `Omit<CreateBaseServerOptions, 'multiprocess'>` — but it's not really worth it at this point.

I acknowledge that I'm losing the hand with the options in the CLI — I hope to refactor this very soon on the airplane to Los Angeles, 12 hours flight.